<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinGameDialog</class>
 <widget class="QDialog" name="JoinGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Join Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeWidget" name="gameListView">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>15</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>State</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Players</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Game Info</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>[Name]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelDescription">
            <property name="text">
             <string>[Description]</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>State:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelState">
            <property name="text">
             <string>[State]</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Players:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="labelPlayers">
            <property name="text">
             <string>[Players]</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>AI Players:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="labelAIPlayers">
            <property name="text">
             <string>[AI Players]</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="labelPassword">
            <property name="text">
             <string>[Password]</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Lifepoints in combinations:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="labelLifePoint">
            <property name="text">
             <string>[Lifepoints]</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>DodgeCity:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="labelDodgeCity">
            <property name="text">
             <string>[DodgeCity]</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>WalleyOfShadows:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="labelWalleyOfShadows">
            <property name="text">
             <string>[WalleyOfShadows]</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>TeamFortress:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLabel" name="labelTeamFortress">
            <property name="text">
             <string>[TeamFortress]</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Our rules:</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLabel" name="labelOurFlag">
            <property name="text">
             <string>[Our rules]</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Stacking dynamite:</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLabel" name="labelStackingDynamite">
            <property name="text">
             <string>[Stacking dynamite]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="playerListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="indentation">
          <number>15</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Players</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Create new player</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Player Name</string>
           </property>
           <property name="buddy">
            <cstring>lineEditPlayerName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditPlayerName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="client::SelectPlayerIconWidget" name="selectPlayerIconWidget">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Avatar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="toolTip">
            <string>You can set your password here. This password will be required to reconnect to your player in case of disconnection.</string>
           </property>
           <property name="text">
            <string>Player Password</string>
           </property>
           <property name="buddy">
            <cstring>lineEditPlayerPassword</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditPlayerPassword">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPlay">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSpectate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Spectate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>client::SelectPlayerIconWidget</class>
   <extends>QLabel</extends>
   <header>selectplayericonwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>JoinGameDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
